lib/signaling/v2/transport.js

```diff
@@ -82,7 +82,10 @@ class Transport extends StateMachine {
     options = Object.assign({
       InsightsPublisher,
       NullInsightsPublisher,
-      SIPJSMediaHandler: DefaultSIPJSMediaHandler
+      SIPJSMediaHandler: DefaultSIPJSMediaHandler,
+      userAgent: typeof navigator !== 'undefined' && navigator.userAgent
```

Comment:
Should this call `util.getUserAgent()`?

Reply:
Oops, I thought I fixed that--good catch